session/bootstrap.go

```diff
@@ -264,6 +264,12 @@ const (
 	CreateExprPushdownBlacklist = `CREATE TABLE IF NOT EXISTS mysql.expr_pushdown_blacklist (
 		name char(100) NOT NULL
 	);`
+
+	// CreateOptRuleBlacklist stores the list of disabled optimizing operations.
+	CreateOptRuleBlacklist = `CREATE TABLE IF NOT EXISTS mysql.opt_rule_blacklist (
+		name char(100) NOT NULL,
+		type char(20) check(type="logical_rule") NOT NULL
```

Comment:
Why add check` since tidb does not support it?

Reply:
It's just a reminder for people. Like code comments, or `show create table mysql.opt_rule_blacklist` will contains it.